kubeslice/docs
# feat(): Footer fixes and updates, logo updated according to cncf project

## What type of PR is this? (check all applicable)

- [x] Upgrades

## Description
- Footer is updated with categorization of links and social icons, with addition of Kubeslice content
- Logos are changed according to the kubeslice cncf project
- Footer in dark mode is fixed
- Huge gap between the top bar and the content is fixed
- Overflow issue is fixed in responsiveness

## QA Instructions, Screenshots, Recordings

- Compare the logos
- Scroll down to footer, to see the new updated change
- Compare the gap between the top bar and the content
- Toggle darkmode on and off
![image](https://github.com/kubeslice/docs/assets/36334914/ad098679-4bd3-4db8-b51f-abb80e33fd5c)

![image](https://github.com/kubeslice/docs/assets/36334914/c6713bb0-7962-4335-ab49-ad8cb8b7c9de)


### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
